boards/posix/native_posix/doc/board.rst

```diff
@@ -185,4 +173,198 @@ with ``valgrind``.
+    :alt: Comparison of different debugging targets
+    :figclass: align-center
+
+    Comparison of different debugging options. Note that realism has many
+    dimensions: Having the real memory map or emulating the exact time an
+    instruction executes is just some of it; Emulating peripherals accurately
+    is another side.
+
+This native port compiles your code directly to x86, with no instrumentation or
+monitoring code. Your code executes directly in the host CPU. That is, your code
+executes just as fast as it possibly can.
+
+Simulated time is decoupled from real host time.
+The problem of how to emulate the instruction execution speed is solved
+by assuming that code executes in zero simulated time.
+
+There is no I/O or MMU emulation. If you try to access memory through hardcoded
+addresses your binary will simply segfault.
+The drivers and HW models for this architecture will hide this from the
+application developers when it relates to those peripherals.
+In general this port is not meant to help developing low level drivers for
+target HW. But for developing application code.
+
+Your code can be debugged, instrumented or analyzed with all normal native
+development tools just like any other Linux application.
+
+Execution is fully reproducible, you can pause it without side-effects.
+
+How does this port compare to QEMU:
+===================================
+
+With QEMU you compile your image targeting the board which is closer to
+your desired board. For example an ARM based one. QEMU emulates the real memory
+layout of the board, loads the compiled binary and through instructions
+translation executes that ARM targeted binary on the host CPU.
+Depending on configuration, QEMU also provides models of some peripherals
+and, in some cases, can expose host HW as emulated target peripherals.
+
+QEMU cannot provide any emulation of execution speed. It simply
+executes code as fast as it can, and let's the host CPU speed determine the
+emulated CPU speed. This produces highly indeterministic behavior,
+as the execution speed depends on the host system performance and it's load.
+
+As instructions are translated to the host architecture, and the target CPU and
+MMU are emulated, there is a performance penalty.
+
+You can connect gdb to QEMU, but have few other instrumentation abilities.
+
+Execution is not reproducible. Some bugs may be triggered only in some runs
+depending on the computer and its load.
+
+How does this port compare to an ISS:
+======================================
+
+With a cycle accurate instruction set simulator you compile targeting either
+your real CPU/platform or a close enough relative. The memory layout is modeled
+and some or all peripherals too.
+
+The simulator loads your binary, slowly interprets each instruction and
+accounts for the time each instruction takes.
+Time is simulated and is fully decoupled from real time.
+Simulations are in the order of x1/10 to x1/100 slower than real time.
```

Comment:
Better would be to say:

     Simulations are on the order of 10 to 100 times slower than real time.